app/src/test/java/mobile/programming/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -32,14 +32,12 @@ class InstrumentDataIntegrationTest {
     }
 
     private fun checkIfInstrumentDataIsEmpty() {
-        var instrumentlist = ArrayList<Instrument>()
+        var instrumentList = ArrayList<Instrument>()
```

Comment:
Noticed that this variable could use some camel casing, as recommended in [https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules](url)

Reply:
A good suggestion again, but covered in a prior review.